torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -393,7 +392,7 @@ llvm::Value* LLVMCodeGen::emitMaskedLoad(
       llvm::BasicBlock::Create(*context_.getContext(), "tail", fn_);
 
   // Test the mask
-  auto cond = irb_.CreateICmpEQ(mask, llvm::ConstantInt::getTrue(int32Ty_));
+  auto cond = irb_.CreateICmpEQ(mask, llvm::ConstantInt::get(int32Ty_, 1));
```

Comment:
@resistor llvm Debug mode was complaining here (initially that getTrue needed an i1 type, then that an i32 couldn't be compared to i1), so I did the minimal thing that passed our tests, but idk if my fix is actually correct since I don't fully "get" the mask semantics.  So take a look when you can :-)

Reply:
Fix looks correct.  Thanks!